Panic on list command when no session is running

```
$ kak -l
$ kks list
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/kkga/kks/kak.(*Session).Dir(0xc000077cf0)
	/home/teddy/src/kks/kak/kak.go:33 +0x8e
github.com/kkga/kks/cmd.(*ListCmd).Run(0xc0001b4320)
	/home/teddy/src/kks/cmd/list.go:75 +0x176
github.com/kkga/kks/cmd.Root({0xc0001a8010, 0x0, 0xc000066750})
	/home/teddy/src/kks/cmd/root.go:38 +0x689
main.main()
	/home/teddy/src/kks/main.go:21 +0xe8
```

Reason: in `kak/kak.go` 

```
strings.Split(strings.TrimSpace(string(o)), "\n")
```
Always returns 1 element slice where the only element is empty string when `o == ""`.


Simple fix:

```diff
diff --git i/kak/kak.go w/kak/kak.go
index 270ff4..3037f7 100644
--- i/kak/kak.go
+++ w/kak/kak.go
@@ -46,6 +48,9 @@ func (s *Session) Clients() (clients []Client, err error) {
 func Sessions() (sessions []Session, err error) {
 	o, err := exec.Command("kak", "-l").Output()
 	for _, s := range strings.Split(strings.TrimSpace(string(o)), "\n") {
+		if s == "" {
+			continue
+		}
 		sessions = append(sessions, Session{s})
 	}
 	return
```

--- a/kak/kak.go
+++ b/kak/kak.go
@@ -1,9 +1,10 @@
 package kak
 
 import (
+	"bufio"
+	"bytes"
 	"errors"
 	"os/exec"
-	"strings"
 )
 
 type Context struct {
@@ -12,9 +13,11 @@ type Context struct {
 	Buffer  Buffer
 }
 
-type Session struct{ Name string }
-type Client struct{ Name string }
-type Buffer struct{ Name string }
+type (
+	Client  struct{ Name string }
+	Session struct{ Name string }
+	Buffer  struct{ Name string }
+)
 
 func (s *Session) Exists() (exists bool, err error) {
 	sessions, err := Sessions()
@@ -45,8 +48,11 @@ func (s *Session) Clients() (clients []Client, err error) {
 
 func Sessions() (sessions []Session, err error) {
 	o, err := exec.Command("kak", "-l").Output()
-	for _, s := range strings.Split(strings.TrimSpace(string(o)), "\n") {
-		sessions = append(sessions, Session{s})
+	scanner := bufio.NewScanner(bytes.NewBuffer(o))
+	for scanner.Scan() {
+		if s := scanner.Text(); s != "" {
+			sessions = append(sessions, Session{s})
+		}
 	}
 	return
 }

